VAMobile/src/screens/PaymentsScreen/DirectDepositScreen/EditDirectDepositScreen/EditDirectDepositScreen.tsx

```diff
@@ -137,6 +137,21 @@ const EditDirectDepositScreen: FC<EditDirectDepositProps> = ({ navigation, route
     return !/^\d+$/.test(input)
   }
 
+  const invalidRoutingNumber = (input: string): boolean => {
```

Comment:
This function says `067014822` is an invalid routing number, even though it's actually valid. I noticed the logic in one of the [links](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/6429#issuecomment-1664242741) that Tim shared resembles the [logic](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/6429#issuecomment-1664406416) that EVSS is using. We'll probably want to use one of those approaches instead

Reply:
@theodur that is the logic I used is the one that Tim and EVSS is using.

The logic is that numbers 1, 4, 7 are multiplied by 3, numbers 2, 5, 8 are multiplied by 7, and finally 3, 6, 9 are multiplied by 1. There was a typo that I was multiplying 1, 4, 7 and 2, 5, 8 by 3. I have fixed that and it now works.